resources/warnings.json

```
@@ -16926,7 +16926,7 @@
     "versions": [
       {
         "lastVersion": "2.5",
-        "pattern": ".*"
+        "pattern": "(1|2[.][0-5])(|[.-].+)"
```

@daniel-beck , should a trailing literal "." be added after the "1" in case the plugin is switched to use automated release?
```suggestion
        "pattern": "(1[.]|2[.][0-5])(|[.-].+)"
```

---

Taken care of by the matching groups.

---

Thanks!